src/sentry/api/endpoints/project_app_store_connect_credentials.py

```diff
@@ -154,7 +154,12 @@ def post(self, request: Request, project: Project) -> Response:
         )
         session = requests.Session()
 
-        apps = appstore_connect.get_apps(session, credentials)
+        try:
+            apps = appstore_connect.get_apps(session, credentials)
+        except appstore_connect.UnauthorizedError:
+            raise AppConnectAuthenticationError()
```

Comment:
python oddity, you don't actually have to instantiate the class you want to "raise".  If you pass a type to `raise` it will instantiate it for you.  But there's no harm in doing this either so doesn't really matter.

Reply:
this is probably the second or third time i've done this in a PR, thanks for pointing this out 🙃